trailmet/algorithms/binarize/birealnet.py

```diff
@@ -0,0 +1,229 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+import os
+import sys
+import shutil
+import numpy as np
+import time, datetime
+import torch
+import random
+import logging
+import argparse
+import torch.nn as nn
+import torch.utils
+import torch.backends.cudnn as cudnn
+import torch.distributed as dist
+import torch.utils.data.distributed
+sys.path.append('/workspace/code/kushagrabhushan/TrAIL/trailmet/trailmet/algorithms/binarize')
```

Comment:
Re-work this and make more generic

Reply:
Sorry, could you elaborate a little more?